plugins/woocommerce/includes/class-wc-download-handler.php

```diff
@@ -270,7 +270,7 @@ public static function parse_file_path( $file_path ) {
 		);
 
 		$count            = 0;
-		$file_path        = str_replace( array_keys( $replacements ), array_values( $replacements ), $file_path );
+		$file_path        = str_replace( array_keys( $replacements ), array_values( $replacements ), $file_path, $count );
```

Comment:
With this, we no longer need to explicitly initialize `$count` to zero (line 272) _but_ I think it's fine/actually makes things a little clearer.